test/modules/lydia/fnModNameConflict.good

```diff
@@ -1,2 +1,4 @@
-fnModNameConflict.chpl:3: error: 'M1' has multiple definitions, redefined at:
-  fnModNameConflict.chpl:9
+fnModNameConflict.chpl:1: In module 'OuterModule':
+fnModNameConflict.chpl:14: error: 'M1' is ambiguous
+fnModNameConflict.chpl:3: note: it could refer to this 'M1'
+fnModNameConflict.chpl:9: note: or it could refer to this 'M1'
```

Comment:
In this case, I feel like the original error was better.  I.e., I think it should be an error to define the same symbol multiple times at a given scope in all cases other than overloading, and would prefer to see the error be an active one characterizing the scope defining the symbols rather than what feels like a more passive one here, only appearing when the incorrectly overloaded symbol is referenced.  Namely, if there were no reference to 'M1', I think we should still get an error (similar to if we defined a var and a const with the same name in a given scope, or a var and a proc, or pretty much anything other than an overloaded proc or overloaded iter).

Reply:
I am planning to improve our dyno multiply-defined symbols errors in https://github.com/Cray/chapel-private/issues/4409 . I have not yet started on that but I plan to look at it soon. I expect that I can get it to emit a multiply-defined symbols error in this case.

Reply:
> Namely, if there were no reference to 'M1', I think we should still get an error (similar to if we defined a var and a const with the same name in a given scope, or a var and a proc, or pretty much anything other than an overloaded proc or overloaded iter).

That is the case today even with `--dyno` (in that mode, the production scope resolver emits the error -- it is still running even though we have already scope-resolved many things with the dyno one).

Reply:
I think I see—so you're saying that an earlier dyno error is preventing us from getting to the later production compiler error, and that once that error is re-implemented in dyno, the old behavior will return (potentially with better worded errors, etc.)?

Reply:
yes, exactly